tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc

```diff
@@ -1036,6 +1036,7 @@ TEST_F( EnergyPlusFixture, UnitarySystem_GetInput ) {
 	Schedule( 1 ).CurrentValue = 1.0;
 	DataGlobals::BeginEnvrnFlag = true;
 	DataEnvironment::StdRhoAir = PsyRhoAirFnPbTdbW( 101325.0, 20.0, 0.0 ); // initialize RhoAir
+	Node( InletNode ).MassFlowRateMaxAvail = UnitarySystem( 1 ).MaxCoolAirVolFlow * StdRhoAir;
```

Comment:
Well then you would use the max flow rate. These are unit tests where heat/cool flow are equal. Are you saying I should have used max(UnitarySystem( 1 ).MaxCoolAirVolFlow, UnitarySystem( 1 ).MaxHeatAirVolFlow, ...) * StdRhoAir here instead for maintenance sake if someone copied one of these unit tests?